test/lib.bash

```diff
@@ -266,7 +265,6 @@ function teardown {
   fi
   logger.warn "Teardown 💀"
   teardown_serverless
-  teardown_tracing
```

Comment:
Why do you delete this one?

Reply:
ah, I wanna add that to the upstream too ....  PR is still on hold :-)